files/Aboot/boot0.j2

```diff
@@ -376,7 +432,7 @@ write_platform_specific_cmdline() {
     local varlog_size=100
 
     # detect the size of the flash partition from name in Aboot/EOS/SONiC
-    local flash_size=$(($(df | grep -E "$flash_re" | tr -s ' ' | cut -f2 -d' ') / 1000))
+    local flash_size=$(($(df | grep "$target_path$" | tr -s ' ' | cut -f2 -d' ') / 1000))
```

Comment:
>df | grep "$target_path$" [](start = 26, length = 25)

Could you just `df $target_path` ?

If this is not working in Aboot, could you `grep " $target_path$"`, ie adding one more space at the beginning of the pattern?

Reply:
`df $target_path` works but still require a pipe with `tail -1` to trim the header off.
will make the change.